lib/kennel.rb

```diff
@@ -63,19 +63,21 @@ def update
 
     def store(parts)
       Progress.progress "Storing" do
-        old = Dir["generated/#{project_filter || "**"}/*"]
-        used = []
-
-        Utils.parallel(parts, max: 2) do |part|
-          path = "generated/#{part.tracking_id.tr("/", ":").sub(":", "/")}.json"
-          used << File.dirname(path) # only 1 level of sub folders, so this is safe
-          used << path
-          payload = part.as_json.merge(api_resource: part.class.api_resource)
-          write_file_if_necessary(path, JSON.pretty_generate(payload) << "\n")
-        end
+        (project_filter || ["**"]).each do |project_pattern|
```

Comment:
this would also work and not need another nesting ?
```
        old = Dir["generated/{#{project_filter || "**"}}/*"]
```

Reply:
```ruby
irb(main):001:0> Dir["generated/#{['kube2iam', 'zeus_smoke']}/*"]
=> []
```

No, I don't think so.